### DISCUSSION TITLE
State Timeline: Support aligning the series name on the left or right

### DISCUSSION THREAD

**Discussion by mellieA** [grafana/grafana#48659] [Dashboards, Panels & Visualizations]
**Current**
The series name in the statetimeline panel is aligned to line up the last character of every series name. 

**Feature request**
This request is to make the alignment of the series name configurable, so a user can choose to line up the first letter of all series names. 

<img width="715" alt="Screen Shot 2022-05-03 at 12 26 16 PM" src="https://user-images.githubusercontent.com/18175287/166518435-48e76df4-7ba1-4cb4-b284-f81cd25e6c30.png">

Community requested in this post: https://community.grafana.com/t/how-format-text-for-metric-in-state-timeline/65049

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
